app/controllers/APIController.scala

```diff
@@ -80,4 +111,29 @@ class APIController @Inject()(etmpService: ETMPService,
     }
     Ok(Json.toJson(responseData))
   }
+
+  private def returnResponseForAPI(penaltyDetails: GetPenaltyDetails, enrolmentKey: String)(implicit request: Request[_]): Result = {
+    val pointsTotal = penaltyDetails.lateSubmissionPenalty.map(_.summary.activePenaltyPoints).getOrElse(0)
+    val penaltyAmountWithEstimateStatus = apiService.findEstimatedPenaltiesAmount(penaltyDetails)
+    val noOfEstimatedPenalties = apiService.getNumberOfEstimatedPenalties(penaltyDetails)
+    val crystallisedPenaltyAmount = apiService.getNumberOfCrystallisedPenalties(penaltyDetails)
+    val crystallisedPenaltyTotal = apiService.getCrystallisedPenaltyTotal(penaltyDetails)
+    val hasAnyPenaltyData = apiService.checkIfHasAnyPenaltyData(penaltyDetails)
+    val responseData: APIModel = APIModel(
+      noOfPoints = pointsTotal,
+      noOfEstimatedPenalties = noOfEstimatedPenalties,
+      noOfCrystalisedPenalties = crystallisedPenaltyAmount,
+      estimatedPenaltyAmount = penaltyAmountWithEstimateStatus,
+      crystalisedPenaltyAmountDue = crystallisedPenaltyTotal,
+      hasAnyPenaltyData = hasAnyPenaltyData)
+    if(pointsTotal > 0) {
```

Comment:
Not one to fix now *unless you want too) but what about if the user has no LSP and only LPP the audit event would not fire.
Looks like this was the same on previous implementation so happy to raise bug if not fixed now

Reply:
Will look into it now

Reply:
Will change to `hasAnyPenaltyData`
```suggestion
    if(hasAnyPenaltyData) {
```

Reply:
👍  Nice yes that is the simplest solution I was thinking we need a few ors in the if statement but yes hasAnyPenaltyData should do what we need